pkg/pillar/scripts/device-steps.sh

```diff
@@ -561,3 +579,9 @@ while true; do
     access_usb
     sleep 300
 done
+
+echo "At the end of device steps"
```

Comment:
This is dead code due to the while true loop. Please remove.

Reply:
Will remove